src/server/middleware/logs.ts

```diff
@@ -1,71 +1,90 @@
-import type { FastifyInstance } from "fastify";
+import type { FastifyInstance, FastifyRequest } from "fastify";
 import { stringify } from "thirdweb/utils";
 import { logger } from "../../shared/utils/logger";
 import { ADMIN_QUEUES_BASEPATH } from "./admin-routes";
 import { OPENAPI_ROUTES } from "./open-api";
 
-const SKIP_LOG_PATHS = new Set([
-  "",
+const IGNORE_LOG_PATHS = new Set([
   "/",
   "/favicon.ico",
   "/system/health",
   "/static",
   ...OPENAPI_ROUTES,
-  // Skip these routes case of importing sensitive details.
+]);
+
+const SENSITIVE_LOG_PATHS = new Set([
   "/backend-wallet/import",
   "/configuration/wallets",
+  "/backend-wallet/lite",
 ]);
 
+function shouldLog(request: FastifyRequest) {
+  if (!request.routeOptions.url) {
+    return false;
+  }
+  if (request.method === "OPTIONS") {
+    return false;
+  }
+  if (
+    request.method === "POST" &&
+    SENSITIVE_LOG_PATHS.has(request.routeOptions.url)
+  ) {
+    return false;
+  }
+  if (IGNORE_LOG_PATHS.has(request.routeOptions.url)) {
+    return false;
+  }
+  if (request.routeOptions.url.startsWith(ADMIN_QUEUES_BASEPATH)) {
+    return false;
+  }
+  return true;
+}
+
 export function withRequestLogs(server: FastifyInstance) {
   server.addHook("onSend", async (request, reply, payload) => {
-    if (
-      request.method === "OPTIONS" ||
-      !request.routeOptions.url ||
-      SKIP_LOG_PATHS.has(request.routeOptions.url) ||
-      request.routeOptions.url.startsWith(ADMIN_QUEUES_BASEPATH)
-    ) {
-      return payload;
-    }
-
-    const { method, routeOptions, headers, params, query, body } = request;
-    const { statusCode, elapsedTime } = reply;
-    const isError = statusCode >= 400;
+    if (shouldLog(request)) {
```

Comment:
Random whitespace changes in this file. I confirmed its using spaces now, so I must have saved it with tabs in the past. (Grrr Biome)